.github/workflows/nightly_build.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         if [ ${{ matrix.compiler }} = gcc ]; then
           export CC=gcc
           export CXX=g++
-          export KRATOS_CMAKE_OPTIONS_FLAGS="-DUSE_EIGEN_MKL=ON -DUSE_EIGEN_FEAST=ON -DMMG_ROOT=/external_libraries/mmg/mmg_5_5_1/"
+          export KRATOS_CMAKE_OPTIONS_FLAGS="-DUSE_EIGEN_MKL=ON -DUSE_EIGEN_FEAST=ON -DMMG_ROOT=/external_libraries/mmg/mmg_5_5_1/ -DPMMG_ROOT=/external_libraries/ParMmg_4d272bb -DINCLUDE_PMMG=ON"
```

Comment:
good you remembered :D
I already forgot...

Reply:
I remembered just because I saw your warning here from the other time it failed :D